docs/api_docs/open_api/3.0.0/swagger.json

```diff
@@ -6,43 +6,5 @@
     "version": "1"
   },
   "paths": {
-    "/api/v1/open_library/{isbn}": {
```

Comment:
why delete this?

Reply:
I think this line was deleted because this branch hasn't some tests, now I merge that branch with the branch development, It's solved